business_logic/db/DBCommunicator.js

```diff
@@ -14,42 +14,51 @@ experimentsCollection.loadExperimentsCollection(database_global)
 actionsCollection.loadActionsCollection(database_global)
 tweetsCollection.loadTweetsCollection(database_global)
 
+//TO BE CHANGED AFTER HACKHATON!!
 function insertExperiment (experiment){
-    experimentsCollection.insertExperiment(experiment)
-    // Call for more collection modules if needed.
+    experimentsCollection.insertExperiment(experiment);
+    //wipe the whole db
+    actionsCollection.deleteActions();
+    participantsCollection.deleteParticipants();
+    researchersCollection.deleteResearchers();
+    tweetsCollection.deleteTweets();
 }
 
 function getExperimentByCode(expCode){
+    return experimentsCollection.getExperimentByCode(expCode);
 
 }
 
 function insertParticipant(participant) {
-    // Insert to experimentsCollection
-    // Insert to participantsCollection
+    var parsedData = JSON.parse(participant);
+    var expId = participant.exp_id; 
```

Comment:
It is better to use "let" instead of var inside a function (prevent problems that are hard to debug)